### PR TITLE
show toast notifications over modals

### LINKED ISSUE #1632: z-index of toast notifications is not high enough to overlap the overlay component
Toast notifications and the overlay component have the same z-index of 1050, we want toasts to pop up over top of the overlay.

z-index

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Fixes #1632

### CODE REVIEW

File: app/styles/_core.less [hunk 1]
```diff
@@ -1235,3 +1235,8 @@ pre.clipped {
     width: 100%;
   }
 }
+
+// Override Patternfly z-index for toast notifications list to keep toasts above modals
+.toast-notifications-list-pf {
+  z-index: @zindex-modal + 5;
```
Comment (spadgett) [inline app/styles/_core.less]: Curious if `+ 5` is arbitrary or if we're using `+ 1`, `+ 2`, `+ 3` ... elsewhere?

**Comment by spadgett** [comment]
Should this go in common?

**Comment by jeff-phillips-18** [comment]
I used +5 as the halfway point between @zindex-modal and @zindex-popover

It could go in common.

**Comment by spadgett** [comment]
> I used +5 as the halfway point between @zindex-modal and @zindex-popover

Makes sense. Do you mind adding that as a comment?

**Comment by jeff-phillips-18** [comment]
Not at all.

**Comment by spadgett** [comment]
@rhamilto @sg00dwin Should the toast be below popover? Trying to understand how that will interact with the dropdowns we have in our masthead.

**Comment by jeff-phillips-18** [comment]
I think it should be, if I interact with something (causing the popover) I would expect it not to be cutoff by the toast.

**Comment by jeff-phillips-18** [comment]
console includes patternfly less files after catalog and common, so any catalog or common overrides will not work. @rhamilto is this what we want or should we re-order?

**Comment by spadgett** [comment]
> I think it should be, if I interact with something (causing the popover) I would expect it not to be cutoff by the toast.

Yeah I think this is probably right, but wanted to ask the question. It think it means the toast will slide in between the menu trigger and the menu itself, but I think that's OK?

We might check that that's really what happens since I know setting `position` on elements changes how `z-index` can behave.

**Comment by rhamilto** [comment]
> console includes patternfly less files after catalog and common, so any catalog or common overrides will not work. @rhamilto is this what we want or should we re-order?

We should reorder if we need to override.  I initially put 'em in there in alpha order.

**Comment by jeff-phillips-18** [comment]
The toast covers both the trigger and the menu. They are childed to the navbar which is specifically meant to be below the toast notifications.

**Pull request closed without merging**